### ISSUE TITLE
Page 9: Slide with too much content

### ISSUE THREAD

**Issue by pht-uol** [NFDI4Energy/EFZN_rdm#51]
It's an important point, but I find the slide very cluttered because there are so many aspects described with text that you can't even remember them all at once.

**Comment by JuliaHeiken** [comment]
Do you mean Legal Aspects?

**Comment by pht-uol** [comment]
Yes, exactly. (It partly holds for the following slide as well.)
